data/valgrind/valgrind-test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash ex
 # SUSE's openQA tests
 #
 # Copyright 2020 SUSE LLC
```

Comment:
This line and the maintainer in line 8 here as well. Sorry I could not select this before you edited the shebang.